internal/logger.go

```diff
@@ -16,36 +16,39 @@ package containersuseconnect
 
 import (
 	"fmt"
+	"io"
 	"log"
 	"os"
 )
 
 // The default path for the logger if nothing has been specified.
-var defaultLogPath = "/var/log/suseconnect.log"
-
-// The environment variable used to specify a custom path for the logger
-// path.
-const logEnv = "SUSECONNECT_LOG_FILE"
-
-// GetLoggerFile returns the output file for the logger. If the `logEnv` environment
-// variable has been set, it will try to output there. Otherwise, it
-// will try to output to the file as given in `defaultLogPath`. If
-// everything fails, it will just output to the standard error channel.
-func GetLoggerFile() *os.File {
-	// Determine the path to be used.
-	var path string
-	if env := os.Getenv(logEnv); env != "" {
-		path = env
-	} else {
-		path = defaultLogPath
+const DefaultLogPath = "/var/log/suseconnect.log"
```

Comment:
Any particular reason why this becomes an exported field now?

Reply:
no real reason to be honest, the package is already internal, and I see no reason for everything being private as well.

being a public (or private) constant no one can change it, being a private variable, some can change its value.